PRIVACY_POLICY.md

```diff
@@ -0,0 +1,65 @@
+# Swarm-Extension Privacy Policy
+
+The Swarm Extension belongs to the Swarm Foundation with the purpose of offering web3 environment
+for running decentralised applications (dApps). It means, it provides similar functionalities as web2 browsers,
+so sensitive information may be handled by the browser-extension such as Bee client configuration
+or cross-domain localStore. Although the extension handles these sensitive information for
+the basic security and functionality of dApps, the Swarm Foundation does not share any of these data
+with other 3rd parties, nor collect it.
```

Comment:
To make it clearer, maybe reword:

"Although the extension handles these sensitive information for
the basic security and functionality of dApps, the Swarm Foundation does not collect this information and therefore also cannot and does not share it with 3rd parties."